web/index.html

```diff
@@ -141,38 +141,38 @@
   margin-top: 3px;
 }
 #nav-container .icon-bar:nth-of-type(1) {
-  background-color: #00ffff;
-  box-shadow: 0px 0px 5px #00ffff, 0px 0px 2px #00ffff;
+  background-color: #00ff00;
```

Comment:
All this is to change the icons when we have mode B selected vs mode 4C. It's so subtle (swapping green/cyan) that I'm not sure that it's worth it. *But* having green up top and magenta at the bottom reflects the color order we're actually using, and I couldn't resist.